CHANGES.rst

```diff
@@ -18,6 +18,9 @@ astropy.coordinates
 - Angle parsing now supports ``cardinal direction`` in the cases
   where angles are initialized as ``string`` instances. eg ``"17°53'27"W"``.[#9859]
 
+- ```SkyCoord.radial_velocity_correction``` no longer raises an Exception when space motion
```

Comment:
This entry should not be in "new feature," but I'll let the subpackage maintainer decides if this is a bug fix (4.0.x) or API change (4.1).